contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-api.mdx

```diff
@@ -0,0 +1,88 @@
+## Step 1: Evaluate the feature flag value using the `/decide`
+
+`/decide` is the endpoint used to determine if a given flag is enabled for a certain user or not.
+
+#### Request
+
+```shell
+curl -v -L --header "Content-Type: application/json" -d '  {
+    "api_key": "<ph_project_api_key>",
+    "distinct_id": "distinct_id_of_your_user",
+    "groups" : { # Optional
```

Comment:
groups are required when you have any group based flags, those wouldn't work without this